src/Voice.tsx

```diff
@@ -578,6 +578,38 @@ export class Voice extends EventEmitter {
     return callInvitesMap;
   }
 
+  /**
+   * Handle Firebase messages from an out-of-band Firebase messaging service.
+   *
+   * @remarks
+   *
+   * Note that this method only works on Android platforms, and will only work
+   * when the built-in Firebase messaging service as been opted-out.
+   *
+   * Unsupported platforms:
+   * - iOS
```

Comment:
Do you want to add a note if this will supported in the future, or if we do not plan to support in the future?

Reply:
It won't be necessary to support in the future, as iOS does not have the same Firebase messaging service limitations as Android does.